stdlib/public/core/CompilerProtocols.swift

```diff
@@ -658,7 +658,7 @@ public protocol ExpressibleByArrayLiteral {
 ///
 ///     let countryCodes = ["BR": "Brazil", "GH": "Ghana",
 ///                         "JP": "Japan", "US": "United States"]
-///     // 'countryCodes' has type [String: String]
+///     // 'countryCodes' has type '[String: String]'
```

Comment:
In this file, types are enclosed in single quotes. To maintain consistency, I have made similar adjustments.
For example: 'integers' has type '[Int]', 'emptyArray' has type '[Bool]'.